tests/sles4sap/redirection_tests/ensa2_hanadb_refresh_sapinstances.pm

```diff
@@ -0,0 +1,88 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Test module tests ENSA2 Central Services with HANA DB - refresh sapinstance ASCS & ERS.
+#   After refresh no failover happens, fail count stays at 0, cluster is healthy.
+
+use parent 'sles4sap::sap_deployment_automation_framework::basetest';
+
+use warnings;
+use strict;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use sles4sap::console_redirection;
+use hacluster qw(check_cluster_state crm_check_resource_location);
+use sles4sap::sap_deployment_automation_framework::deployment qw(load_os_env_variables get_sdaf_instance_id);
+use saputils;
+
+sub execute_crm_resource_refresh_and_check {
```

Comment:
why this function is better to be implemented in test module and not in a lib?

Reply:
> why this function is better to be implemented in test module and not in a lib?

I will move it to lib.

Reply:
Moved to lib and added UT, new VR is good: https://openqaworker15.qa.suse.cz/tests/322357#step/ensa2_hanadb_refresh_sapinstances/95